Source/KSCrash/Source/KSCrash/Recording/Sentry/BSG_KSCrashSentry_User.c

```diff
@@ -96,34 +93,12 @@ void bsg_kscrashsentry_reportUserException(const char *name, const char *reason,
         BSG_KSCrash_Context *reportContext = bsg_kscrashsentry_generateReportContext();
         BSG_KSCrash_SentryContext *localContext = &reportContext->crash;
 
-        // We want these variables to persist until the onCrash
-        // call later
-        int callstackCount = 100;
```

Comment:
I'm unclear what this function is now doing now - does it need to suspend threads?